perma_web/frontend/lib/store.js

```diff
@@ -12,14 +13,57 @@ export const getLinksRemainingStatus = (linksRemaining, isNonpaying) => {
     return globalStore.updateLinksRemainingStatus('unlimited_paid')
 }
 
-export const getUserTypes = (isIndividual) => {
-    let types = []
-
+export const getUserTypes = (isIndividual, user) => {
     if (isIndividual) {
-        types = types.concat('individual')
+        return globalStore.updateUserTypes('individual')
+    }
+
+    let userTypes = []
+
+    const topLevelFolders = user.top_level_folders
+    const isOrgAffiliated = topLevelFolders.some(folder => !!folder.organization)
+    const isSponsored = topLevelFolders.some(folder => folder.is_sponsored_root_folder)
```

Comment:
Same, let's pass this info straight to the JS, rather than calculating it from the folders.

Reply:
Also refactored in https://github.com/harvard-lil/perma/commit/9b238bb1bc24b2404c7950532c6837ce224fd403